open-api/rest-catalog-open-api.yaml

```diff
@@ -1464,19 +1464,23 @@ components:
     LoadTableResult:
       description:
         Result used when a table is successfully loaded.
-        
-        
-        The table metadata JSON is returned in the `metadata` field. The corresponding file location of table metadata must be returned in the `metadata-location` field. Clients can check whether metadata has changed by comparing metadata locations.
-        
-        
+
+
+        The table metadata JSON is returned in the `metadata` field. The corresponding file location of table metadata should be returned in the `metadata-location` field, unless it hasn't been determined yet - such as when a transaction begins to stage a table for creation but has not commit.
```

Comment:
Instead of adding ` - ` and extending the sentence, start a new one with "For example".

It should be something like "unless the metadata is not yet committed. For example, a create transaction may return metadata that is staged but not committed."